spec/bourbon/addons/text_inputs_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "spec_helper"
+
+describe "text-inputs" do
+  before(:all) do
+    ParserSupport.parse_file("addons/text-inputs")
+
+    @inputs_list = [
+      'input[type="color"]',
+      'input[type="date"]',
+      'input[type="datetime"]',
+      'input[type="datetime-local"]',
+      'input[type="email"]',
+      'input[type="month"]',
+      'input[type="number"]',
+      'input[type="password"]',
+      'input[type="search"]',
+      'input[type="tel"]',
+      'input[type="text"]',
+      'input[type="time"]',
+      'input[type="url"]',
+      'input[type="week"]',
+      'textarea'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.